saleor/order/models.py

```diff
@@ -233,6 +236,23 @@ def __iter__(self):
             return iter(self.lines.all())
         return super().__iter__()
 
+    @transition(
+        field=status, source=GroupStatus.NEW, target=GroupStatus.NEW)
+    def process(self, cart_lines, discounts=None):
+        process_delivery_group(self, cart_lines, discounts)
+
+    @transition(
+        field=status, source=GroupStatus.NEW, target=GroupStatus.SHIPPED)
+    def ship(self):
```

Comment:
Shouldn't the tracking number be a parameter passed to `ship(...)`?

Reply:
The only place where we are using ship method for now is in ```ShipGroupForm```, which already has `tracking_number` field. I'm not sure if we should move it here.

Reply:
I believe assigning the tracking number is part of the shipping action we're modeling here and thus belongs in the transition.